lib/pages/image_viewer/image_viewer.dart

```diff
@@ -58,12 +67,43 @@ class ImageViewerController extends State<ImageViewer> {
     }
   }
 
+  void toggleShowMoreActions() {
+    if (menuController.isOpen) {
+      menuController.close();
+    } else {
+      menuController.open();
+    }
+  }
+
+  void showInChat() {
+    if (!PlatformInfos.isMobile) {
+      handleShowInChatInWeb();
+    } else {
+      handleShowInChatInMobile();
+    }
+  }
+
+  void handleShowInChatInWeb() {
+    Navigator.of(context).pop();
+    widget.onBack?.call();
+    if (widget.event != null) {
+      context.goToRoomWithEvent(widget.event!.room.id, widget.event!.eventId);
+    }
+    return;
+  }
```

Comment:
IMO: Should separate the funcs to make it clearer.

- `Navigator.of(context).pop()` how to use?
- `widget.onBack?.call()` what is this? how to use it?

Reply:
IMO: Should separate func two case: 

1. Web desktop (Not close Chat Details)
2. Web tablet (Close Chat Details)